src/printer.js

```diff
@@ -2256,6 +2256,12 @@ function isEmptyJSXElement(node) {
   }
 }
 
+function isJSXWhitespace(node) {
+  if (node.type !== "if-break") return false;
+  if (node.flatContents !== " ") return false;
+  return (node.breakContents ===  '{" "}' || node.breakContents === "{' '}");
```

Comment:
We really shouldn't be accessing the internal properties of the printer nodes. If you're looking to do `doc === jsxWhitespace`, you can create the `jsxWhitespace` doc in `printJSXElement` and pass it into `printJSXChildren`.